core/domain/stats_domain.py

```diff
@@ -1351,3 +1352,115 @@ def validate(self):
             raise utils.ValidationError(
                 'calculation_output is too big to be stored (size: %d): %s' % (
                     sys.getsizeof(output_data), str(output_data)))
+
+
+class LearnerAnswerDetails(object):
+    """Domain object that represents the answer details submitted by the
+    learner.
+    """
+
+    def __init__(
+            self, state_reference, entity_type, interaction_id,
+            learner_answer_info_list, schema_version=(
+                feconf.CURRENT_LEARNER_ANSWERS_DETAILS_SCHEMA_VERSION)):
+        """Constructs a LearnerAnswerDetail domain object.
+
+            Args:
+                state_reference: str. The reference to the state.
+                entity_type: str. The type of entity, eg. exploration,
+                    question etc.
+                interaction_id: str. The ID of the interaction.
+                learner_answer_info_list: list. The list of LearnerAnswerInfo
+                    objects and version specified in this object.
+                schema_version: str. The schema version of this object.
+        """
+        self.state_reference = state_reference
+        self.entity_type = entity_type
+        self.interaction_id = interaction_id
+        self.learner_answer_info_list = learner_answer_info_list
+        self.schema_version = schema_version
+
+    def get_learner_answer_info_dict_list(self):
+        """Returns the learner_answer_info_list stored within this object as
+        a list of LearnerAnswerInfo dicts.
```

Comment:
Missing Returns part of docstring.

(Please try and do a self-review before requesting a review, to catch issues like this.)

Reply:
Done